composer.json

```diff
@@ -13,12 +13,12 @@
     }
   ],
   "require": {
-    "monolog/monolog": "^1.25",
+    "monolog/monolog": "2.5.*",
```

Comment:
wouldn't `^2.5` be a little more idiomatic?

Reply:
You're right :+1: